docs/updating/from_1.x_2.x/update_app_to_2.5.md

```diff
@@ -13,16 +13,16 @@ latest_tag: '2.5.27'
 
 [[% include 'snippets/update/merge_composer.md' %]]
 
-## 3. Update the app
+## 3. Version Specific changes
 
-First, perform version-specific steps depending on which version you are updating from.
+When updating a release **earlier than v2.5.0**, some version-specific changes are required.
```

Comment:
@reithor, in general, can you explain the reason for this change? We use the same structure for all update pages (see https://doc.ibexa.co/en/master/updating/from_3.3/update_from_3.3/#v332 for example), so changing it here would require reformulating the headings everywhere in the update section for consistency.
If I understand correctly and the issue is: "we are updating **to** v2.2, not **from** v2.2", I'd prefer rephrasing this introductory sentence:
```suggestion
First, perform version-specific steps depending on which version you are updating to.
```

Reply:
@DominikaK : changing the structure is not neccessary - I just thought it woul be clearer.
The whole point of the PR is that :
"A. v2.2" --> should be "A. from earlier than v2.2"
"B. v2.5" --> should be "B. from earlier than v2.5"